frontend/src/scalars/ui/Config.vue

```diff
@@ -18,31 +24,32 @@
             <v-radio label="Wall" value="wall"></v-radio>
         </v-radio-group>
 
-        <label class="label">Tooltip sorting method</label>
```

Comment:
We need to display this Label.

Reply:
v-selector has this label embedded

Reply:
I think we need to null the v-model in order for the label to show up then.